demo/index.html

```diff
@@ -25,6 +25,8 @@
       --searchResultText: #adaedc;
       --searchResultBg: #272958;
       --searchResultBorder: #fff;
+      --closeMenuIconWidth: 18px;
```

Comment:
suggestion to make this 20px, from designs.

Reply:
Ah, I thought I had it right from eyeballing it. Updated.